packages/core/src/messages/responses/cache-set-contains-elements.ts

```diff
@@ -0,0 +1,103 @@
+import {SdkError} from '../../errors';
+import {
+  BaseResponseError,
+  BaseResponseMiss,
+  ResponseBase,
+} from './response-base';
+import {truncateString} from '../../internal/utils';
+import {CacheSetContainsElementsResponse} from './enums';
+
+const TEXT_DECODER = new TextDecoder();
+
+interface IResponse {
+  /**
+   * Returns a mapping of the elements to their presence in the cache.
+   * @returns {Record<string, boolean> | undefined} A mapping of the elements to their presence in the cache.
+   * If the set itself was not found, (ie the response was a `Miss` or `Error`), this method returns `undefined`.
+   */
+  containsElements(): Record<string, boolean> | undefined;
```

Comment:
does record not support `Record<string | Uint8Array, boolean>`? If the user wants to find if a specific binary element was a hit, they will have to know to use a textdecoder to decode the binary element first and then do a lookup

Reply:
I modelled this off of `dictionaryGetFields` (see [here](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/core/src/messages/responses/cache-dictionary-get-fields.ts)).

There are similar concerns: a user can request various fields from a dictionary. We give the response back as a record for convenience. The fields can be string vs Uint8Array and so can the values. We have accessors there for each combination, but we assume the most common request is for string-string.

Here I am assuming the most common case is the set contains strings. We can add other accessors if users request. Aiming to keep the initial API slim.